### DISCUSSION TITLE
Add id in form()->open()

### DISCUSSION THREAD

**Discussion by bulfaitelo** [spatie/laravel-html#188] [Q&A] [answered]
Hello, I've been using laravel-html, because I've been migrating from Laravel Collective, I'm really enjoying it, but I'm having some problems.

I need to add an id to the form to be able to apply some validations, but I'm not getting it, is there any way to inject an id into the `html()->form()`? to get a result similar to this:
`<form method="POST" action="[action/id]"  id="formAbc">`

Note that I added `id="formAbc"`, I wanted a way to be able to inject this id when creating the form to be able to use validators

**Comment by fabricio-fn** [comment, marked as answer]
you can use attributes() method

{!! html()->form('POST', '[action/id]')->attributes(['id' => 'formAbc']) !!}
    ...
{!! html()->form()->close() !!}

**Reply by bulfaitelo** [reply]
Ty!

**Reply by kamleshwebtech** [reply]
Thanks a lot dear, it worked for me, you saved my life :)

**Comment by levaconline** [comment]
{!! html()->form('POST', '[action/id]')->id('formAbc') !!}
